packages/matter.js/src/behavior/system/network/ServerNetworkRuntime.ts

```diff
@@ -25,6 +25,13 @@ import { CommissioningBehavior } from "../commissioning/CommissioningBehavior.js
 import { SessionsBehavior } from "../sessions/SessionsBehavior.js";
 import { NetworkRuntime } from "./NetworkRuntime.js";
 
+type NetworkInterfaceDetails = {
```

Comment:
WDYT about adding an optional "interface type" field here and converting the return value from Network.getNetInterfaces() to a list of objects that also includes the interface type?

If we then leave a TODO in NetworkNode it would allow users to override NetworkNode to add their own implementation (or submit a PR).

Might be worthwhile to provide a bare-bones linux-only implementation that looks for /sys/class/net/(interface name)/wireless to differentiate between ethernet and wifi.

Reply:
What do you think abot that idea ...

Reply:
Lgtm